frontend/src/components/AppHeader.tsx

```diff
@@ -20,7 +20,10 @@ export default function AppHeader({}) {
     try {
       // Try to show the magic wallet UI
       // This will only work if the user has connected via a magic wallet, not via browser wallet (e.g. MetaMask)
-      await magic.wallet.showUI();
+      /// @ts-ignore
```

Comment:
2 things:
1. What's the error that we're ignoring with @ts-ignore
2. Is showUI an async method? If so we should await it right?